app/controllers/api/v1/book_suggestion_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Api
+  module V1
+    class BookSuggestionController < ApplicationController
+      def create
+        User.find(params[:user_id]) if params[:user_id].present?
```

Comment:
if not present continue normally, and why make the user query if not using the data

Reply:
to catch exception in case the user is not found

Reply:
but the `user_id` not present continue without problem, you should manage with a conditional 
```ruby
render json: { error: 'bla bla' }, status: : unprocessable_entity unless params[:user_id].present?
```

Reply:
because when `user_id` is not present I consider it as an guest user since it is not necessary to be authenticated according to the card https://trello.com/c/BJopZ710/25-1-crear-ruta-para-creaci%C3%B3n-de-un-booksuggestion

Reply:
yes sounds good, but why find a User if you don't use the data is better to delete the line, but if you set this in the params to create a book suggestion make sense and is better

Reply:
because if I add a user_id that is not in my database it generates an exception but maybe it will improve it using pundit